pkg/syncthing/process.go

```diff
@@ -14,12 +14,37 @@
 package syncthing
 
 import (
+	"runtime"
 	"time"
 
 	"github.com/shirou/gopsutil/process"
 )
 
 func terminate(p *process.Process, wait bool) error {
+	if runtime.GOOS == "windows" {
+		children, err := p.Children()
```

Comment:
how long does it take this call?

Reply:
I have run the code for 5 times and the average time to call for the children was 10.2ms (been 3.8 the fastest and 14.9 the slowest)